addons/get_info/tests/test_get_info.py

```diff
@@ -0,0 +1,123 @@
+# -*- coding: utf-8 -*-
+# from . import common
+from odoo.tests import common
+from ..models.models import BASIC_GET_INFO_VALUES
+from collections import Counter
+
+
+@common.at_install(False)
+@common.post_install(True)
+class TestStockLocation(common.SavepointCase):
+    @classmethod
+    def setUpClass(cls):
+        super(TestStockLocation, cls).setUpClass()
+        cls.Location = cls.env["stock.location"]
+        # Get stock location
+        cls.stock_location = cls.Location.search([]).filtered(lambda l: l.name == "Stock")
```

Comment:
Why don't we use cls.env['XXX']?

Reply:
Using the cls.Location above. But only use Location once elsewhere, so tidied this up